### PR TITLE
Complete Result page layout

### PR DESCRIPTION
**Pull request by poludnev**

### CODE REVIEW

File: src/lib/apiClient.ts [hunk 1]
```diff
@@ -64,7 +64,10 @@ export const fetchPreviewDataByTweetId = async (
 export const submitNotarization = async (
   tweetId: string,
   cb?: (data: any) => void,
-): Promise<ContractReceipt | { status: 'failed' | 'success'; error?: string | null }> => {
+): Promise<
+  // TODO: make relevant type, extend ContractReceipt with error status
```
Comment (utgarda) [inline src/lib/apiClient.ts:68]: Pls add 
* who this TODO is for, like  `// TODO @poludnev ...`
* follow-up task ID like  `// TODO @poludnev #123 relevant type`
How do you feel about a task for this kind of `TODO` items?

File: src/pages/Preview/Preview.tsx [hunk 2]
```diff
@@ -40,11 +40,14 @@ export const Preview: React.FC<IPreviewProps> = () => {
             <TweetResults imageUrl={data.imageUrl} tweetdata={data.tweetdata} tweetId={tweetId} />
             <MetadataPreview
               blocked={!data.metadata}
-              title={`Http headers meta${!data.metadata && ': metadata unavailable'}`}
+              title={`Http headers meta${!data.metadata ? ': data unavailable' : ''}`}
```
Comment (utgarda) [inline src/pages/Preview/Preview.tsx:43]: `Http headers meta ${`
Probably wants and extra space

File: src/pages/Preview/Preview.tsx [hunk 3]
```diff
@@ -40,11 +40,14 @@ export const Preview: React.FC<IPreviewProps> = () => {
...
             >
               {data.metadata && <HTTPMetadata metadata={data.metadata} />}
             </MetadataPreview>
-            <MetadataPreview blocked={!data.metadata} title='DNS headers meta'>
+            <MetadataPreview
+              blocked={!data.metadata}
+              title={`DNS headers meta${!data.metadata ? ': data unavailable' : ''}`}
```
Comment (utgarda) [inline src/pages/Preview/Preview.tsx:49]: .

**Review by utgarda — APPROVED** [review]
LGTM
Please take care of cosmetic issues and merge

File: src/pages/Preview/Preview.tsx (hunk 2 above)
Comment (poludnev) [reply src/pages/Preview/Preview.tsx:43]: that extra space is just right here

File: src/pages/Preview/Preview.tsx (hunk 3 above)
Comment (poludnev) [reply src/pages/Preview/Preview.tsx:49]: false: DNS headers meta: data unavailable
true: DNS headers meta

**Pull request merged**